README.md

```diff
@@ -131,10 +131,10 @@ Copyright (c) Microsoft Corporation. All rights reserved.
 
 Licensed under the [Source EULA](LICENSE.txt).
 
-[win-user]: https://go.microsoft.com/fwlink/?linkid=2142210
-[win-system]: https://go.microsoft.com/fwlink/?linkid=2142135
-[win-zip]: https://go.microsoft.com/fwlink/?linkid=2142211
-[osx-zip]: https://go.microsoft.com/fwlink/?linkid=2142136
-[linux-zip]: https://go.microsoft.com/fwlink/?linkid=2142212
-[linux-rpm]: https://go.microsoft.com/fwlink/?linkid=2142213
-[linux-deb]: https://go.microsoft.com/fwlink/?linkid=2142214
+[win-user]: https://go.microsoft.com/fwlink/?linkid=2144429
```

Comment:
there shouldn't be new fwlinks for a hotfix release.  the main point of the fwlinks is they always point to the latest release for the month.

Reply:
I didn't know about this and made new fwlinks for this release. Should we still go ahead with it since it's published? I'll put down this as a note in the OneNote for future hotfixes.

Reply:
You should probably still update the old fwlinks as well